website/blog/2019-09-23-linux-foundation.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Presto now hosted under the Linux Foundation
+author: Ariel Weisberg
+authorURL: https://www.linkedin.com/in/ariel-weisberg-a5b6899
+authorFBID: 100006183888902
+---
+
+We are happy to announce that today we have launched the Presto Foundation, a non-profit organization under the umbrella of the Linux Foundation, together with our partners Alibaba, Twitter, and Uber!
```

Comment:
I would put the collaboration more up front:

> We are excited to announce today, in partnership with Alibaba, Twitter and Uber, the launch of the Presto Foundation, a non-profit organization under the umbrella of the [Linux Foundation](https://www.linuxfoundation.org/).

You might also want to link to Twitter, Uber and Alibaba as well.

Reply:
Did what you suggested with a slight change. I added Facebook to the list of partners. Listed companies in alphabetical order.